brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1037 Fix: Call To Action - CTA break when click on button tab from style.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1037
- Issue is due to the line-height option is not being used in the button typography component for CTA.

### Screenshots

- Fix - https://d.pr/v/vf4yZq

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the CTA block.
- Go to style > Click on the button tab.
- Check is it working fine or not.

### Checklist:
